discovery/eureka_test.go

```diff
@@ -0,0 +1,119 @@
+package discovery
+
+import (
+	"gopkg.in/ory-am/dockertest.v3"
+	dc "github.com/fsouza/go-dockerclient"
+	"github.com/jaume-pinyol/fargo"
+	"log"
+	"os"
+	"testing"
+	"strconv"
+	"github.com/op/go-logging"
+)
+
+var eurekaTestPort int = 8080
+var eurekaTestUrl string = "http://127.0.0.1:" + strconv.Itoa(eurekaTestPort) + "/eureka"
+
+func TestMain(m *testing.M) {
+	// uses a sensible default on windows (tcp/http) and linux/osx (socket)
+	pool, err := dockertest.NewPool("")
```

Comment:
why are you using "dockertest" instead of "dockertest.v3"? has the dot no sense?

Reply:
http://labix.org/gopkg.in

Reply:
👍